### ISSUE TITLE
Title not extracted correctly and wrongly mistaken for excess

### ISSUE THREAD

**Issue by datagutt** [jzjzjzj/parse-torrent-name#84]
Filename: ````[720pMkv.Com]_sons.of.anarchy.s05e10.480p.BluRay.x264-GAnGSteR.mp4````
Output:
````
    { season: 5,
      episode: 10,
      resolution: '720p',
      quality: 'BluRay',
      codec: 'x264',
      group: 'GAnGSteR',
      title: '[',
      excess: 'Mkv.Com]_sons.of.anarchy.480p' } 
````

Filename: ``[ www.Speed.cd ] -Sons.of.Anarchy.S07E07.720p.HDTV.X264-DIMENSION.mp4``
Output:
````
 { season: 7,
      episode: 7,
      resolution: '720p',
      quality: 'HDTV',
      title: '[ www.Speed.cd ] -Sons.of.Anarchy.'} 
````

**Comment by datagutt** [comment]
This is how i chose to fix it:
https://github.com/Seriesbox/seriesbox-server/blob/ca7df4bea8a6ef84b7ca551568fb8b9b1b824b80/src/parsers/tv/Parser.js#L39

**Comment by jzjzjzj** [comment]
Released v0.5.4 with the [fixes](https://github.com/jzjzjzj/parse-torrent-name/compare/v0.5.3...v0.5.4).

**Comment by datagutt** [comment]
:+1:
